Jenkinsfile-publish

```diff
@@ -0,0 +1,32 @@
+pipeline {
+  agent none
+
+  parameters {
+    choice(name: "VERSION", choices: ["patch", "minor", "major"], description: "The stage you are building the front end for")
+  }
+  stages {
+    stage("Publish package") {
+        agent {
+            ecs {
+                inheritFrom "npm"
+            }
+        }
+        
+        steps {
+            checkout scm
+            sh 'npm ci'
+            sh 'npm run build:prod'
+            sh 'git config --global user.email tna-digital-archiving-jenkins@nationalarchives.gov.uk'
```

Comment:
I can't remember which repo it was, but is this the same email address that you talked to David about last week? Because I don't think this email address exists: Jenkins uses some pre-existing DRI address.

Reply:
I did ask him but all we've really got is a distribution list for TDR which isn't really what we're after. This will do as a placeholder, especially as we don't want to encourage people on npm to email the whole team, and anyone who legitimately wants to get in touch can do via github.

Reply:
I've [added it to the backlog](https://github.com/orgs/nationalarchives/projects/20#card-34783403).